asserter/asserter.go

```diff
@@ -147,6 +186,9 @@ func NewClientWithFile(
 		config.AllowedOperationStatuses,
 		config.AllowedErrors,
 		&config.AllowedTimestampStartIndex,
+		&Validations{
+			Enabled: false,
```

Comment:
Could you add a comment why we do not need validation config here?

Reply:
`rosetta-cli` doesn't use this function. I tried to limit this functionality to that so that's why disabled it.